packages/react-sdk/package.json

```diff
@@ -62,9 +63,10 @@
     "react-nano-state": "^0.4.0",
     "zod": "^3.17.3"
   },
-  "main": "src/index.ts",
```

Comment:
This is the same as [here](https://github.com/webstudio-is/webstudio-designer/pull/163/files#diff-38b40d8cc858fc034768ee58216c9fdb6f15f82e4c1b38d3e1114ccab19e1c17).